src/mlagility/analysis/analysis.py

```diff
@@ -113,6 +113,11 @@ def call_benchit(
             build_name=build_name,
             cache_dir=cache_dir,
             build_only=Action.BENCHMARK not in tracer_args.actions,
+            lean_cache=tracer_args.lean_cache,
+            groq_num_chips=tracer_args.num_chips,
+            groq_compiler_flags=tracer_args.compiler_flags,
+            groq_assembler_flags=tracer_args.assembler_flags,
+            groqview=tracer_args.groqview,
```

Comment:
Update `user_guide.md` to use `groq_num_chips` instead of `num_chips` and so on.